kentikapi/internal/utils/utility_functions.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+import (
+	"encoding/json"
+	"log"
+)
+
+func LogPayload(l bool, msg string, payload interface{}) {
+	if l {
+		if payload == "" {
+			log.Println(msg)
+		} else {
+			jsonData, err := json.Marshal(&payload)
+			if err != nil {
+				log.Fatal(err)
```

Comment:
Rule of thumb: library should not call log.Fatal() / os.Exit() / panic().
SDK operation should not fail because of log formatting either.
Please just use payload instead of jsonData in log.Println() if Marshal fails.

Reply:
Done